src/app/containers/Timestamp/relativeTimestamp.test.js

```diff
@@ -0,0 +1,61 @@
+import relativeTimestamp from './relativeTimestamp';
+
+const timeStampGenerator = (timeDifference, magnitude) =>
+  Date.now() - timeDifference * magnitude;
+
+const timestampWithDiffInHours = hours =>
+  timeStampGenerator(hours, 60 * 60 * 1000);
+
+const timestampWithDiffInMins = mins => timeStampGenerator(mins, 60 * 1000);
+
+const relativeBehaviour = (description, input, expectedOutput) => {
+  it(description, () => {
+    const result = relativeTimestamp(input);
+    expect(result).toEqual(expectedOutput);
+  });
+};
+
+describe('relativeTimestamp', () => {
```

Comment:
Probably not needed, but can we have a test for in the future to ensure we catch the edge case

Reply:
![roll safe meme](https://media.giphy.com/media/d3mlE7uhX8KFgEmY/giphy.gif)
too smart 👍

Reply:
but actually.. what would the expected behaviour be for a case where the timestamp is in the future? 
the expected behaviour for more than 10 hours in the past is to show an absolute timestamp but we wrote this thinking that case would be caught before this function is called.
would we want a future timestamp to be caught before this is called?
@dr3

Reply:
Probably one for @jimjohnsonrollings, im more just concerned it doesnt break if someone has their device clock set wrong

Reply:
Why would there be a future timestamp?

Reply:
@jimjohnsonrollings Someone might have their phone clock set wrong, such that the device thinks the ares timestamp is in the future

Reply:
How do we handle this on responsive?  I have a vague memory that we don't use the system clock - but send our server time as part of the payload to use as 'now'.

Reply:
For the record, looks like we've decided to go with an empty string for the future: https://github.com/bbc/simorgh/pull/1380/commits/4ce9feee0cdb04b1b234f9e355b4cddc6829a11a

That feels odd to me - what gets announced to screen readers?

Reply:
https://github.com/bbc/simorgh/pull/1380/commits/a61a7f3fa50d9e35f38dd5613aaca13e8d4aa146 we're returning null now,

From what I understand, instead of a relative timestamp the absolute timestamp will be displayed instead.